taccsite_cms/templates/guides/getting_started.html

```diff
@@ -91,12 +90,36 @@ <h3 id="create-account">Creating an Account</h3>
     </div>
   </div>
 
+  <div class="row">
+    <div class="col">
+      <h3 id="request-access-before-login">Requesting Access</h3>
+
+      <p>
+        Some projects require an allocation or other access be provided before login is possible.
+      </p>
+
+      <div class="row">
+        <div class="col">
+          <p>
+            <i class="icon icon-multiple-coversation icon-multiple-conversation icon--as-image"></i>
+          </p>
+        </div>
+
+        <div class="col">
+          <p>
+            If you are unable to login, please inquire whether your project requires allocation before login, and if so request that allocation from an appropriate member.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
I could use a script to show this only for certain portals, but I still hope for a generic static document.

<details>A JavaScript solution could be simple and inline, but I'd probably go overboard and create a reusable module. A Python script would work for users who disable JavaScript, but I am slow writing Python.</details>